label_studio_converter/converter.py

```diff
@@ -410,17 +411,17 @@ def convert_to_csv(self, input_data, output_dir, is_dir=True, **kwargs):
             if 'agreement' in item:
                 record['agreement'] = item['agreement']
             records.append(record)
+            keys.update(list(record.keys()))
```

Comment:
why do we need this variable?

Reply:
Different tasks can have different keys, to have all of them in export we need a complete list